easybuild/easyconfigs/v/VarScan/VarScan-2.3.6-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'JAR'
+
+name = 'VarScan'
+version = '2.3.6'
+
+homepage = 'http://varscan.sourceforge.net/'
+description = """Variant calling and somatic mutation/CNV detection for next-generation sequencing data"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [('%(name)s.v%(version)s.jar', 'download')]
+source_urls = ['https://sourceforge.net/projects/varscan/files/']
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
```

Comment:
@verdurin please use `%(javaver)s`:

```python
name = 'VarScan'
version = '2.3.6'
versionsuffix = '-Java-%(javaver)s'
...
dependencies = [('Java', '1.7.0_80')]
```

you can then just the `java` and `javaver` "local variables"